Scripts/Handles/BaseHandle.cs

```diff
@@ -106,6 +106,9 @@ public void OnBeginDrag(RayEventData eventData)
 				return;
 
 			var rayOrigin = eventData.rayOrigin;
+			if (!this.IsConeVisible(rayOrigin) || !this.IsRayVisible(rayOrigin))
+				return;
```

Comment:
Why do this in BaseHandle, not MultipleRayInputModule?

Reply:
Added IBlockUIInteraction check in AddRaycastSource valid check (in EditorVR.Rays).